Checkout v201 docs (#398)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/tutorials/add-payment-method.mdx

## Step-by-step

> This tutorial provides a step-by-step guide for integrating the Braintree payment gateway provider, as an example, with the Commerce boilerplate template. You can use these steps as a reference to integrate other payment providers as well.

Reviewer: ```suggestion
This tutorial walks you through integrating Braintree as a payment provider with the Commerce boilerplate template. While we use Braintree as an example, you can adapt these same steps for other payment providers.
```

<Aside type="note">
Make sure that the payment provider you choose offers an SDK or API that you can use to integrate with the [slots](/dropins/checkout/slots/) provided by the [`PaymentMethods`](/dropins/checkout/containers/payment-methods/) container in the checkout drop-in component.
</Aside>

<Aside type="note">

> The _**commerce-checkout.js**_ block is the only one that is fully functional and up-to-date with the latest version of the checkout drop-in component.

Reviewer: ```suggestion
The `commerce-checkout.js` block is the only fully functional implementation that's up-to-date with the latest checkout drop-in component. 
```

> Use the following guidelines just as a reference when creating a new checkout experience.

Reviewer: ```suggestion
Use these steps as a reference when developing your custom checkout experience.
```

   let braintreeInstance;
   ```

> 1. Update the [`PaymentMethods`](/dropins/checkout/containers/payment-methods/) container to include a custom handler for the Braintree payment method and set `autoSync` to `false` to prevent automatic calls to the [`setPaymentMethod`](/dropins/checkout/functions/#setpaymentmethod) function on change.

Reviewer: ```suggestion
1. Update the [`PaymentMethods`](/dropins/checkout/containers/payment-methods/) container to include a custom handler for the Braintree payment method. Set `autoSync` to `false` to prevent automatic calls to the [`setPaymentMethod`](/dropins/checkout/functions/#setpaymentmethod) function when the payment method changes.
```
